erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -117,6 +124,9 @@ def set_picked_qty(self):
 		for row in self.locations:
 			if not row.picked_qty:
 				row.picked_qty = row.stock_qty
+			
+			if row.picked_qty > row.stock_qty:
+				frappe.throw(_("Row #{0} : Picked Qty should be less or equal to Stock Qty").format(frappe.bold(row.idx), frappe.bold(row.stock_qty)))
```

Comment:
I don't see you using the `row.stock_qty` format option

```suggestion
				frappe.throw(_("Row #{0}: Picked Qty should be less or equal to Stock Qty").format(frappe.bold(row.idx), frappe.bold(row.stock_qty)))
```